src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area,Drawable {
```

Comment:
According to the checklist, abstract classes should not be used to set behavior for classes. Consider using an interface to define behavior and abstract classes for shared state or functionality if necessary.